src/lib/viewers/controls/color-picker/ColorPickerPalette.scss

```diff
@@ -0,0 +1,46 @@
+.bp-ColorPickerPalette {
+    position: absolute;
+    left: -55px;
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 35px;
```

Comment:
This is a little confusing. Is the palette itself only 35 pixels tall?

Reply:
The portion of the palette that doesn't include the arrow on the bottom should be 35px tall.

Reply:
We should allow the content to determine the size of the container. Can you post a screenshot?

Reply:
![Screen Shot 2020-12-02 at 5 13 06 PM](https://user-images.githubusercontent.com/7213887/100950675-b41c0600-34c1-11eb-88d3-569a07c38caa.png)

Reply:
Ah I see what you mean. I can just apply some margin to the top / bottom of the swatches so we do not need the 35px defined.